pkg/dom0-ztools/rootfs/etc/init.d/000-mod-params

```diff
@@ -73,4 +73,5 @@ if ! test -f /proc/vmcore; then
     modprobe pinctrl-upboard      2>/dev/null
     modprobe leds-upboard         2>/dev/null
     modprobe xhci_tegra           2>/dev/null
+    modprobe rtc_pcf85063         2>/dev/null
```

Comment:
This was a WIP change. Is this module available on x86? I remember that I did this change because the module was not loading automatically from the device tree... if it's not being loaded, maybe would be better to turn this support built-in in the arm64 kernel.